jadx-gui/src/main/java/jadx/gui/utils/NLS.java

```diff
@@ -6,9 +6,14 @@
 public class NLS {
 
 	private static ResourceBundle messages;
+	private static final boolean IS_CHINESE = Locale.SIMPLIFIED_CHINESE==Locale.getDefault();
 
 	static {
-		load(new Locale("en", "US"));
+        if(IS_CHINESE){
+            load(Locale.SIMPLIFIED_CHINESE);
```

Comment:
Use a better check here

Reply:
@liach can you be more specific how to check locale, where are a lot of ways to do it and I don't know which is better

Reply:
I think we can just load the default locale with a fallback